src/scripts/stockage.js

```diff
@@ -33,22 +33,15 @@ export default class StockageLocal {
     }
 
     supprimerTout() {
-        return localforage
-            .dropInstance()
-            .then(() => {
-                console.debug('Les données personnelles ont été supprimées')
-            })
-            .catch((error) => {
-                console.error(
-                    `Erreur lors de la suppression de toutes les données personnelles`
-                )
-                console.error(error)
-            })
+        return this.unsetProfilActuel().then(() =>
+            this.getProfils().then((noms) =>
+                Promise.all(noms.map((nom) => this._supprimer(nom)))
```

Comment:
```suggestion
                Promise.all(noms.map(this._supprimer))
```

Si ça se trouve ça marche 😅

Reply:
Mais carrément !